Add information about  Dynamic Metadata Service. (#419)
deNBI/cloud-user-docs: wiki/Compute_Center/Bielefeld.md

Openstack API. In this case users have to contact the cloud site administrators in order for them to set 
an explicit local password.

## Instance Metadata

Openstack presents configuration information to instances it starts via a mechanism called metadata.
This metadata can be accessed querying `http://169.254.169.254/`. Services like cloud-init make use 
of this metadata to initialize and configure a started instance.

Beside metadata information in AWS compatible format, Openstack additionally supports metadata in its
own style. There are three different kind of metadata which can be accessed by the user.

| Typ | Description                                                                                   |
|-----|-----------------------------------------------------------------------------------------------|

> | Compute data | Structured data containing information about network, hostname, public-key, ...               |
> | User data | The user has the ability to pass unstructured data like shell scripts, ...  to the instance.  |
> | Vendor data | Optional the cloud provider can make vendor specific information (static or dynamic) availabe |

Reviewer: ```suggestion
| Compute data | Structured data containing information about network, hostname, public-key, etc. |
| User data | The user has the ability to pass unstructured data like shell scripts, etc. to the instance. |
| Vendor data | Optionally the cloud provider can make vendor specific information (static or dynamic) available. |
```

> On request project specific information like all users with their elixir id/name and public-keys can be made

Reviewer: ```suggestion
On request, project specific information like the list of users including their Elixir ID/name and public keys can be made
```

> available. For example, this information can be used to build up a multi-user instance, giving all users of the 

Reviewer: ```suggestion
available. This information could e.g. be used to set up a multi-user instance, giving all users of the 
```

project access.

```
> curl http://169.254.169.254/openstack/latest/vendor_data2.json | jq
>{
"denbi": [
{
"elixir_name": "...",
"id": "...",
"name": "...",
"perun_id": "...",
"public_keys": [
"ecdsa-sha2-nistp256 AAAA...",
]
},
...
]
}
```

> **Important: This feature is not active as a default setting and will only be enabled on request.** 

Reviewer: ```suggestion
**Important: This feature is not active by default but will only be enabled on request.** 
```
